src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  width: 7 * ($day-size + $gap) - $gap + 2 * $calendar-padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $calendar-padding;
+
+  &__day {
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid #000;
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition-duration: 0.5s;
+
+    @for $i from 1 through $default-month-length {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
```

Comment:
The 'transition-duration' property is redundant here since it's already declared in the base state of '&__day' (line 16). You should remove it to avoid confusion and duplication.